### ISSUE TITLE
A11Y : Webform Confirmation Message

### ISSUE THREAD

**Issue by jcsparks** [CuBoulder/tiamat-theme#938]
There are several different way that the Webform module allows users to show a successful form submission message.  One or more of these will place the message either in place of the webform or on the same page. 

For these implementations we need a mechanism to alert screen readers that the confirmation message has been added to the page.  I believe that we can solve this by adding an `aria-live="polite"` to the area where the message will show up.

**Comment by timurtripp** [comment]
I'm unable to start on this as Ajax form submissions aren't working this way for me, they're instead redirecting to a `/form/form-name/confirmation?token=...` page to show the submission confirmation. @patrickbrown-io confirmed this behavior as well and opened CuBoulder/tiamat10-profile#132

**Comment by patrickbrown-io** [comment]
See https://github.com/CuBoulder/tiamat10-profile/issues/132 , may need additional customization to prompt different submission methods

**Comment by timurtripp** [comment]
Thanks for looking into that @patrickbrown-io! Webforms are confusing.

@jcsparks I don't know if this is necessary. The "Message" option places an alert on the page with `role="alert"`. This is [equivalent to `aria-live="assertive"`](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA19). The only potential issue is that it doesn't exist on the page until after the form is submitted. The same is true with the `aria-live="polite"` we added to the people list which only gets appended once someone submits a filter. But [this documentation](https://devdoc.net/web/developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role.html) is making me think the screen reader would call out the alert correctly on the Webform as it is now. Did the accessibility team say otherwise?

**Comment by timurtripp** [comment]
I definitely have some questions about how the `aria-live` attribute works. If the accessibility team found a problem with this due to it being added dynamically, we may need to go back and update the people list again so the `aria-live="polite"` is always present, not just when they submit the filters.

**Comment by timurtripp** [comment]
I also don't know if putting a `role-"alert"` inside an `aria-live="polite"` after the form is submitted will have the intended effect of announcing the alert or not, or if we can assign a less forceful role to the alert since it's just a conformation message (`role-"alert"` is intended primarily for serious errors it seems like).
